src/scripts/modules/media/footer/footer.coffee

```diff
@@ -39,6 +39,7 @@ define (require) ->
     selectTab: (e) ->
       $tab = $(e.currentTarget)
       @switchTab($tab)
+      false
```

Comment:
If this is being used to prevent the browser's default behavior, please use an explicit `e.preventDefault()`.  While returning false does have that affect in jQuery, it means a bit more, and can be less clear when reading the code.